public/css/style.css

```diff
@@ -1,6 +1,8 @@
+
```

Comment:
More weird spacing in the style.css file - specifically lines 1 and 110
Also a few spaces needed after : in line 39 for example, so just have a look through the file again to fix any of those
Also needs a new line at the end of file